wit/testdata_test.go

```diff
@@ -486,11 +486,15 @@ func TestHasBorrow(t *testing.T) {
 	err := loadTestdata(func(path string, res *Resolve) error {
 		t.Run(path, func(t *testing.T) {
 			for _, td := range res.TypeDefs {
-				wit := td.Kind.WIT(nil, "")
-				a := strings.Contains(wit, "borrow<")
-				b := HasBorrow(td)
-				if a != b {
-					t.Errorf("has \"borrow<\": %t != HasBorrow(td): %t (%s)", a, b, wit)
+				// skipping anonymous types
+				// see https://github.com/bytecodealliance/wasm-tools-go/issues/167
+				if td.Name != nil {
```

Comment:
Can you change this to an early out instead of a nested if?

// Skip anonymous types here. This is tested elsewhere.

Reply:
Done.